src/credential-store/credential-store-identity-response.ts

```diff
@@ -4,6 +4,10 @@ export type CredentialStoreIdentityResponse = {
   afterKey?: string;
 };
 
+export type CredentialStoreErrorResponse = {
```

Comment:
We've already a type that is meant to be this in the file `credential-store-error-response.ts` though it is called `ErrorResponse` which is less descriptive.

Can you reuse what's already there rather than introducing a new type? Though renaming the existing one to be `CredentialStoreErrorResponse` rather than just `ErrorResponse` would make sense.